### DISCUSSION TITLE
Having trouble reducing the loss

### DISCUSSION THREAD

**Discussion by mahmudh254** [lululxvi/deepxde#1750] [Q&A]
Hi,

I am trying to solve the following 2-equation system (PDE) for U (output) and V (output) where x(1D space) and t(time)is the input. I have used DeepXDE but the model can't decrease the loss but rather increases to NAN. The boundary condition for U is 1+0.01*sin(2*pi*freq*t) at x = 0. The initial condition (t = 0) for U is 1 and V is 0. Could you please kindly give me any suggestions on where should I look into? Thank you. I have attached the screenshot of the code.

![Capture](https://github.com/lululxvi/deepxde/assets/170659848/9bdf4abd-f417-48c1-a9a0-678d1a03a8b0)
![Capture1](https://github.com/lululxvi/deepxde/assets/170659848/b233d5a9-e86e-4378-bc3f-34c3d56d4f32)
![Capture2](https://github.com/lululxvi/deepxde/assets/170659848/b2a58d5b-99b8-4e99-b773-1c41fb6e162b)
![Capture3](https://github.com/lululxvi/deepxde/assets/170659848/56d89bf4-2a9f-41be-8577-0556fb7859c7)
![Capture4](https://github.com/lululxvi/deepxde/assets/170659848/f754cb61-052d-4b9c-bb8a-f88c3cdcec11)
![Capture5](https://github.com/lululxvi/deepxde/assets/170659848/69e1837c-86ac-4c94-b697-96bb7233bb96)
![Capture6](https://github.com/lululxvi/deepxde/assets/170659848/08ec9f88-3faf-442b-ac34-6d8621a8d4fc)
![Capture7](https://github.com/lululxvi/deepxde/assets/170659848/c75ac478-08db-41ee-b103-632069d643a5)
![Capture8](https://github.com/lululxvi/deepxde/assets/170659848/86785ba2-a613-4a6b-ab11-bc240889fc55)

**Comment by lululxvi** [comment]
See FAQ.
